tests/runtests.py

```diff
@@ -321,6 +321,12 @@ class ActionSelenium(argparse.Action):
     Validate the comma-separated list of requested browsers.
     """
     def __call__(self, parser, namespace, values, option_string=None):
+        # Check if selenium is not installed.
+        try:
+            import selenium  # noqa
+        except ImportError:
+            print("Selenium needs to be installed to run selenium tests.")
+            sys.exit(1)
```

Comment:
Maybe 
```suggestion
    # check if selenium is installed.
    import selenium  # noqa
```
This then crashes with following error message which I think is good enough to work out what's going on. I'm not sure we need a custom error message. 🤔 
```ModuleNotFoundError: No module named 'selenium'```